build/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM centos:centos7
 
 ENV MLNX_OFED_RHEL7_LIBS "https://linux.mellanox.com/public/repo/mlnx_ofed/latest/rhel7.7/x86_64/UPSTREAM_LIBS/"
-ENV MLNX_OFED_VERSION "50100.0"
+ENV MLNX_OFED_VERSION "50218"
```

Comment:
Move this before the FROM
ARG  MLNX_OFED_VERSION. 
and the build command should be with 
docker build --build-arg MLNX_OFED_VERSION=<VERSION> -t ovs-forwarder:<VERSION>  
So the build.sh script should get the OFED_VERSION that you want to build and then it will pass it as arg to Dockerfile and create the propre tag